manage.py

```diff
@@ -454,7 +466,8 @@ def field_counts(name):
     """)
     db.commit()
 
-    selected_collections = db.pluck('SELECT id FROM selected_collections')
+    selected_collections = db.pluck('SELECT collection_id FROM summaries.selected_collections WHERE schema=%(schema)s',
+                                    {"schema": name})
```

Comment:
Maybe extract this to a small method since it is run in two locations (only difference is ORDER BY).